bin/cli.js

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
+  [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '--reporter <value>',        'Reporter name'],
+  [ '--log-level <value>',       'Log level'],
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error':   [ 'LOG_ERROR', 'errors-only' ],
+  'warn':    [ 'LOG_WARN', 'minimal' ],
+  'info':    [ 'LOG_INFO', 'normal' ],
+  'debug':   [ 'LOG_DEBUG', 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
+
+const isLogAtLeast = (level1, level2) => {
+  const value1 = logLevels.indexOf(level1);
+  const value2 = logLevels.indexOf(level2);
+  if (value1 === -1) level1 = 'disable';
+  if (value2 === -1) level2 = 'disable';
+  return value1 >= value2;
+};
```

Comment:
Why not just:
```js
const isLogAtLeast = (level1, level2 = 'disable') =>
  logLevels.indexOf(level1) >= logLevels.indexOf(level2);
```

Reply:
@SemenchenkoVitaliy ping

Reply:
But what if `level1` or `level2` is not correct? As for me, I think that throwing an error as it is now is better than having `-1` and working with it.